Cargo.lock

```diff
@@ -43,6 +43,17 @@ dependencies = [
  "subtle",
 ]
 
+[[package]]
+name = "ahash"
+version = "0.7.8"
```

Comment:
There is already an ahash with version 0.8.12. Let's see if we can use that instead